### PR TITLE
Add intro

### PR DESCRIPTION
**Pull request by Zhang-Y-J**
Add an intro section (title and a paragraph)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
 # [Sample Repo] PR Practice
 A sample repo for practicing how to create Pull Requests
+
+# Introduction
```
Comment (ZhangYijiong) [inline README.md]: Good. I like it.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,7 @@
...
+Creating Pull Requsts (PRs) is needed when using RCS in a multi-person projects. This repo can be used to practice creating PRs.
+
```
Comment (ZhangYijiong) [inline README.md]: Can you write more?

File: README.md (hunk 1 above)
Comment (Zhang-Y-J) [inline README.md:4]: Good.

File: README.md (hunk 2 above)
Comment (Zhang-Y-J) [inline README.md:6]: Can you write more?

**Review by Zhang-Y-J — COMMENTED** [review]
something

**Pull request merged**